### ISSUE TITLE
Resource Report - 1.1.1 Non-Text Content - A linked image is missing alternative text

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#10242]
As a user with a visual impairment, I expect images which convey relevant information to have associated alt text which give a textual description of the image.

**Linked Work Item:**
#9663 

**Expected Result:**
Any images, if they convey information which is useful for the user to understand the content of the page, should contain alt text which is read out by the screen reader.

**Actual Result:**
The linked image in the Resource Report (while not actually visible on the page in this instance) does not have any alternative text, meaning that a visually impaired user will not be able to access important visual information when an image is placed there.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/dd348674-e507-4d82-9d89-1956b7fedc43)

**Comment by imogenroberts** [comment]
**Possible accessibility statement caveat needed for this bug:** provision exists in the site for users to add in their own alt text when uploading an image to the Resource Report, but the alt text field is not compulsory, meaning some users may choose to upload their images to the site without alt text.

**Comment by apeters** [comment]
this is fixed in the latest with the caveat that @imogenroberts mentioned above
